cerberus/assetlist.json

```diff
@@ -20,7 +20,8 @@
         "symbol": "CRBRUS",
         "logo_URIs": {
           "png": "https://cerberus-chain.s3.us-east-2.amazonaws.com/main/logo.png"
-        }
+        },
+        "coingecko_id": "cerberus"
```

Comment:
```suggestion
        "coingecko_id": "cerberus-2"
```

Reply:
Needs to be the coingecko API id, which is not the same as the url name

For cerberus, there's another coin with the api id `cerberus`, so the cosmos one is `cerberus-2`.  Its confusing.


<img width="372" alt="CleanShot 2022-04-02 at 17 46 42@2x" src="https://user-images.githubusercontent.com/979241/161402327-8368c2c0-7d9d-432d-94ce-dc32b0fc8bb3.png">

Reply:
oh wow, thanks! noted for the future, and PR updated :)